src/App.tsx

```diff
@@ -28,7 +28,7 @@ export class App extends React.Component<{}, State> {
       <div className="App">
         <p className="App__message">
           {condition
-            ? 'The last pressed key is [Enter]'
+            ? `The last pressed key is [${condition}]`
```

Comment:
change condition to ${this.state.key} or even better destruct state in your render and use just ${key}